conceptnet5/vectors/query.py

```diff
@@ -173,24 +151,17 @@ def _match_prefix(self, term, prefix_weight):
 
     def expand_terms(self, terms, limit_per_term=10, oov_vector=True):
```

Comment:
It looks like `limit_per_term` is no longer necessary, since it was only used in `_find_neighbors`.